airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java

```diff
@@ -531,18 +538,14 @@ private Database getDatabase(PostgreSQLContainer db) {
     return Databases.createPostgresDatabase(db.getUsername(), db.getPassword(), db.getJdbcUrl());
   }
 
-  private Set<String> listStreams(Database database) throws SQLException {
+  private Set<String> listStreamNames(Database database) throws SQLException {
     return database.query(
         context -> {
           Result<Record> fetch =
               context.fetch(
                   "SELECT tablename, schemaname FROM pg_catalog.pg_tables WHERE schemaname != 'pg_catalog' AND schemaname != 'information_schema'");
           return fetch.stream()
-              .map(record -> {
-                final String schemaName = (String) record.get("schemaname");
-                final String tableName = (String) record.get("tablename");
-                return schemaName + "." + tableName;
-              })
+              .map(record -> (String) record.get("tablename"))
```

Comment:
This isn't perfect since it's pulling out table name and discarding schema information. However felt this is 'good enough' for our current test set up. Lmk if people think this should be more robust.

Reply:
yeap. i was actually thinking about this last night. let me play around with this

Reply:
I felt we would get confused with native Pair, so decided to emulate on with a private class.